lib/tasks/projects.rake

```diff
@@ -194,4 +194,21 @@ namespace :projects do
       end
     end
   end
+
+  desc 'Verify Go Projects'
+  task :verify_go_projects, [:before_date, :limit] => :environment do |_task, args|
```

Comment:
I thought going by created_at date would be the most sensible way of scrolling through the projects since the older ones are more likely to be incorrect.